MaxText/configs/base.yml

```diff
@@ -164,6 +164,7 @@ dataset_name: 'c4/en:3.0.1'
 eval_dataset_name: 'c4/en:3.0.1'
 eval_split: 'validation'
 per_device_batch_size: 12.0
+expansion_factor_real_data: -1 # if -1 then all hosts will load real data
```

Comment:
Nit: Could you detail this comment to say something like "-1 then all hosts will load real data >0 subset of hosts load real data and share with other hosts"?
This would help future users understand when to enable this feature.